Radzen.Blazor/RadzenDropDown.razor

```diff
@@ -13,7 +13,7 @@
         <div class="rz-helper-hidden-accessible">
             <input disabled="@Disabled" aria-haspopup="listbox" readonly="" type="text" tabindex="-1"
                    name="@Name" value="@(internalValue != null ? internalValue : "")" id="@Name"
-                   aria-label="@(!Multiple && internalValue != null ? internalValue : EmptyAriaLabel)" @attributes="InputAttributes"/>
+                   aria-label="@(!Multiple && internalValue != null ? internalValue : EmptyAriaLabel)" @attributes="InputAttributes" autocomplete="@InputAutoCompleteType"/>
         </div>
```

Comment:
This input is hidden and the user cannot enter anything.

Reply:
ah good call. I will remove the InputFilter config hookups.